packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -258,6 +258,11 @@ export const getStyles = (props: IPanelStyleProps): IPanelStyles => {
           ['@supports (-webkit-overflow-scrolling: touch)']: {
             maxHeight: windowHeight
           },
+          '@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none)': {
+            // For IE high contrast mode
+            borderLeft: `1px solid ${palette.neutralLight}`,
```

Comment:
@betrue-final-final per your [comment](https://github.com/OfficeDev/office-ui-fabric-react/pull/7155#issuecomment-479558719) on #7155, should this border be any thicker?

Reply:
I was having the same thought from looking at the screenshot in the description  😄

Reply:
@betrue-final-final says 3px